case_studies/redshift/redshift_from_img/encoder/variational_dist.py

```diff
@@ -0,0 +1,9 @@
+from bliss.catalog import BaseTileCatalog
+from bliss.encoder.variational_dist import VariationalDist
+
+
+class RedshiftVariationalDist(VariationalDist):
+    def sample(self, x_cat, use_mode=True):
+        fp_pairs = self._factor_param_pairs(x_cat)
```

Comment:
I don't think we need this class any longer

Reply:
Since I predict redshifts without predicting other variables like fluxes, locs, I need to have this class to return as BaseTileCatalog instead of the TileCatalog.

Reply:
Got it